package.json

```diff
@@ -28,8 +28,8 @@
 		"./internal/jsdoc/template/publish": "./lib/processors/jsdoc/lib/ui5/template/publish.cjs"
 	},
 	"engines": {
-		"node": "^16.18.0 || >=18.12.0",
-		"npm": ">= 8"
+		"node": "^20.11.0 || >=21.2.0",
+		"npm": ">= 10"
```

Comment:
Do we have a need to raise this? If it's technically not require from our side, I would rather not change it. Especially as npm 10 has some bugs and regressions.

Reply:
Node v20 ships with npm v10. A user would need to actively downgrade to an older version of npm. So I'm not sure who would benefit from allowing older versions.

Even the latest Node v18 ships with npm v10: https://nodejs.org/en/about/previous-releases

Reply:
Alright, then let's go with `>= 10`